src/Microsoft.Data/DataFrame.BinaryOperators.cs

```diff
@@ -16,339 +16,682 @@ public partial class DataFrame
         {
             return df.Add(value);
         }
+        
+        public static DataFrame operator +(bool value, DataFrame df)
+        {
+            return df.ReverseAdd(value);
```

Comment:
I probably should have caught this months ago, but we have a chance of a null reference exception here if `df` is null. This also is possible in the existing code above. Maybe we can just log a separate bug for it and fix it later.

ex.

```
DataFrame df = null;
df2 = 1 + df;
```

I'm not entirely sure on what the behavior here should be. If we should throw an exception, or return `null`. Thoughts?

Reply:
We should throw an exception. Filed https://github.com/dotnet/corefxlab/issues/2776